### PR TITLE
Dev v1

### PR DESCRIPTION
**Pull request by SAYUK09**
Please review and provide feedback

### CODE REVIEW

File: src/pages/Liked-Videos/Liked-Videos.jsx [hunk 1]
```diff
@@ -5,15 +5,22 @@ import { Link } from "react-router-dom";
...
 export function LikedVideos() {
   const { state, dispatch } = useVideos();
+  const { auth } = useAuth();
 
   useEffect(() => {
     (async function () {
       try {
         const response = await axios.get(
-          "https://vid-lib-backend.sayuk.repl.co/likedvideos"
+          "https://vid-lib-backend.sayuk.repl.co/likedvideos",
```
Comment (appsplash99) [inline src/pages/Liked-Videos/Liked-Videos.jsx]: you could've abstracted urls into a specific files and have used like BASE_URL/likevideos

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -7,11 +7,42 @@ import {
...
   useNavigate,
 } from "react-router-dom";
+import { useAuth } from "../../contexts/authContext";
 
 export function Login() {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [error, setError] = useState("");
+  const { auth, setAuth } = useAuth();
+  console.log(auth, "lalal");
```
Comment (appsplash99) [inline src/pages/Login/Login.jsx]: if not used, console logs can be removed

File: src/pages/signup/signup.jsx [hunk 3]
```diff
@@ -14,6 +14,30 @@ export function Signup() {
...
+      const response = await axios.post(
+        "https://vid-lib-api-forked.sayuk.repl.co/register/signup",
+        {
+          name: name,
+          email: email,
+          password: password,
+        }
+      );
+
+      // if (!response.data.User) {
```
Comment (appsplash99) [inline src/pages/signup/signup.jsx]: please remove comments, if not needed.

File: src/App.js [hunk 4]
```diff
@@ -13,16 +18,18 @@ import { Sidebar } from "./components/Sidebar/Sidebar";
...
 import { Login } from "./pages/Login/Login";
 import { useToast } from "./contexts/toastContext";
+import { useAuth } from "./contexts/authContext";
 
 export default function App() {
   const { state, dispatch } = useVideos();
   const { ToastContainer } = useToast();
+  const { auth } = useAuth();
 
   useEffect(() => {
```
Comment (appsplash99) [inline src/App.js:16]: you could have also used a loader(via state) to show a spinner-loader in the ui when the products are being fetched...

File: src/components/Nav/Nav.jsx [hunk 5]
```diff
@@ -36,9 +27,18 @@ export function Nav() {
             <CgPlayListAdd />
```
Comment (appsplash99) [inline src/components/Nav/Nav.jsx:27]: icons could have been bigger when keeping accessibility in mind.

File: src/pages/signup/signup.jsx [hunk 6]
```diff
@@ -15,10 +17,12 @@ export function Signup() {
   const [name, setName] = useState("");
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
```
Comment (appsplash99) [inline src/pages/signup/signup.jsx:19]: when the user types the password in form fiedls,
make sure you hide the text input.

**Review by appsplash99 — COMMENTED** [review]
The code structure and readability is great - everything is extracted its separate folder,
but i see a room for development 💪 in the space of UI - like the sidebar could have been set to display - none in mobile view.

Above all - great job and keep coding 👍

File: src/App.js (hunk 4 above)
Comment (SAYUK09) [reply src/App.js:16]: Looks much better now!
Thanks.

File: src/pages/signup/signup.jsx (hunk 6 above)
Comment (SAYUK09) [reply src/pages/signup/signup.jsx:19]: Thanks for spotting that out

File: src/pages/Login/Login.jsx (hunk 2 above)
Comment (SAYUK09) [reply src/pages/Login/Login.jsx]: Yeah, need to do that in each file

**Pull request merged**